integration-tests/code-generator/build.gradle.kts

```diff
@@ -0,0 +1,25 @@
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
+
+plugins {
+  alias(libs.plugins.kotlin.jvm)
+  alias(libs.plugins.kotlin.kapt)
+  id("conventions.minimal")
+}
+
+dependencies {
+  api("com.squareup.anvil:compiler-api")
+  implementation("com.squareup.anvil:compiler-utils")
+
+  compileOnly(libs.auto.service.annotations)
+  kapt(libs.auto.service.processor)
+
+  testImplementation(testFixtures("com.squareup.anvil:compiler-utils"))
+  testImplementation(libs.junit)
+  testImplementation(libs.truth)
+}
+
+tasks.withType(KotlinCompile::class).configureEach {
```

Comment:
Looks like you could use the new conventions extension here

Reply:
Thanks!  I missed this one...